scripts/deployOldUpgradeableVersion.js

```diff
@@ -59,14 +76,85 @@ module.exports.deployOldColonyVersion = async (contractName, interfaceName, impl
     const OldInterface = contract(JSON.parse(interfaceArtifact));
     OldInterface.setProvider(web3.currentProvider);
 
-    return OldInterface;
+    const oldAuthorityArtifact = fs.readFileSync(`./colonyNetwork-${versionTag}/build/contracts/ColonyAuthority.json`);
+    const OldAuthority = contract(JSON.parse(oldAuthorityArtifact));
+    OldAuthority.setProvider(web3.currentProvider);
+
+    colonyDeployed[interfaceName] = colonyDeployed[interfaceName] || {};
+    colonyDeployed[interfaceName][versionTag] = { OldInterface, OldAuthority, resolverAddress: colonyVersionResolverAddress };
+
+    return colonyDeployed[interfaceName][versionTag];
   } catch (e) {
     console.log(e);
     return process.exit(1);
   }
 };
 
-async function deployOldUpgradeableVersion(contractName, interfaceName, implementationNames, versionTag) {
+module.exports.downgradeColony = async (colonyNetwork, colony, version) => {
+  if (!colonyDeployed.IMetaColony[version]) {
+    throw new Error("Version not deployed");
+  }
+  const accounts = await web3.eth.getAccounts();
+  const editableColony = await getColonyEditable(colony, colonyNetwork);
+
+  const oldAuthority = await colonyDeployed.IMetaColony[version].OldAuthority.new(colony.address, { from: accounts[0] });
```

Comment:
Isn't `accounts[0]` the default here? Does this need to be made explicit?

Reply:
It was not, because I was not setting up the default parameters when creating the truffle contract objects in this file. I am now, so can be removed from here (and the test file that was using them from your other comment).